package.json

```diff
@@ -17,6 +17,10 @@
     "pnpm": "8.6.2"
   },
   "engine-strict": true,
+  "volta": {
+    "node": "18.15.0",
+    "pnpm": "8.6.2"
+  },
```

Comment:
I personally use https://github.com/volta-cli/volta and I am sure other devs do as well.

Since ferdium has strict engine requirements, its good to have this in place and let volta fetch the correct versions of node and pnpm when running e.g. `pnpm install`